Client.xcodeproj/project.pbxproj

```diff
@@ -14430,7 +14454,7 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/ecosia/ios-core.git";
 			requirement = {
-				branch = main;
+				branch = "MOB-1807_route_traffic_to_bing_ab_test";
```

Comment:
I guess we can already merge core and change this to main, right?

Reply:
Happy to revert. Will wait for manual testing inputs first though